corehq/motech/fhir/json-schema/r4/ResourceList.schema.json

```diff
@@ -295,7 +295,7 @@
           "$ref": "QuestionnaireResponse.schema.json#/definitions/QuestionnaireResponse"
         },
         {
-          "$ref": "ReferralRequest.schema.json#/definitions/ReferralRequest"
```

Comment:
Curious how this was wrong, doesn't this file come directly from FHIR?

Reply:
Yes it does! This looks like a bug. "ReferralRequest" was its name in older versions of FHIR. I haven't logged an issue yet, but it's on my to-do list.